.changeset/silver-houses-speak.md

```diff
@@ -0,0 +1,18 @@
+---
+'@commercetools-frontend/application-shell-connectors': major
+'@commercetools-backend/eslint-config-node': major
+'@commercetools-frontend/mc-dev-authentication': major
+'@commercetools-frontend/eslint-config-mc-app': major
+'@commercetools-frontend/babel-preset-mc-app': major
+'@commercetools-frontend/application-config': major
+'@commercetools-frontend/application-shell': major
+'@commercetools-frontend/mc-html-template': major
+'@commercetools-backend/express': major
+'@commercetools-backend/loggers': major
+'@commercetools-frontend/create-mc-app': major
+'@commercetools-frontend/mc-scripts': major
+'@commercetools-frontend/codemod': major
+'@commercetools-frontend/cypress': major
+---
+
+Drop support for Node.js `v14` and `v16`. Make sure you use Node.js `>=v18`.
```

Comment:
I was wondering if we actually want to drop `v16` support now when officially it will be supported until September. As a developer, I would expect `v16` to work everywhere (at least) until official end of support.

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/3039/commits/9dadd6d962e7d9d39f6b7cfdddb599da3bbdb8ab